modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,41 @@
+[%header,cols="40a,50a,50a"]
+|===
+| Feature | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
+| Policy resources | External dependencies, such as libraries or resources cannot be included in the policy and must be available in the runtime | Policies can package resources, eliminating the dependency with the runtime. |
+| Policies performance | Partial non-blocking processing implemented for policies | All policies implement a non-blocking processing strategy |
+| Versioning | Policies do not support proper versioning | Versioning is supported and enforced |
+| Policy execution order | Throttling, Rate Limiting, and CORS policies executed first. Other policies can be reordered | All policies, except CORS, can be reordered 
+| Resource Level Policy Support | Restricted to RAML-based APIs | Support extended to HTTP-based API, along with RAML-based APIs 
+| Class loader isolation  | No class loader isolation supported | Class loader isolation exists between application libraries, extensions, and runtime libraries 
+| Policy change | Throttling | Added the Spike Control policy, which uses an algorithm based on the sliding window mechanism to focus exclusively on reducing spikes in traffic and protecting a Mule Runtime instance. 
+| Authentication policies | HTTP Basic Authentication policy depended on the Simple Security Manager or LDAP Security Manager policies | Simple Basic Authentication and LDAP Basic Authentication policies include Security Manager configuration
+| Rate Limiting policy | The quota is shared between nodes for configured clusters. The policy can share the quota between cluster nodes. Quota not configurable. | * Provides the ability to configure an identifier for better flexibility. Capabilities that can be optionally configured include:
+* Ability to return headers (Expose Headers checkbox)
+* Ability to Clusterize the quota 
+| Policy configuration | Header propagation occurs be default  | * Header propagation no longer occurs by default. * Header propagation can be configured for Token Enforcement, Rate Limiting, and other default policies that set information in headers and provide the capability to propagate it.
+| Custom Policies | Support to build custom policy templates based on XML schema language | * Powerful engine for building custom policies, using the XML schema language
+* improved error handling
+* Support for all Mulesoft components
+* Isolation between policies and application
+* Can be applied to outbound requests, coming back from the backend services
+* Can package additional libraries
+* Support for versioning is now included as policies are uploaded to Exchange. 
+* Better isolation between policies and the application
+|===
+
+The implementation and user experience for custom policies differs considerably in versions Mule 3 and Mule 4. For more information about custom policies based on the Mule version you are using, see the respective Custom policies section for your version.
```

Comment:
I would omit the first sentence.
The second sentence needs clarification: for more info about custom policies, see the Custom section for your version? Do you mean in our doc somewhere? Is there a separate custom topic in some Mule versioned doc?

Reply:
As provided in See Also. Thanks!

Reply:
Yes, there is a specific documentation section for Mule 3 custom policies and another one for Mule 4 custom policies. 

Custom policies are one of the biggest differences between mule 3 and mule 4 and it will impact customers migrating their custom policies from one version to the other for instance (that is why the first sentence)